server/src/main/java/org/apache/iotdb/db/mpp/plan/analyze/ExpressionUtils.java

```diff
@@ -76,19 +60,16 @@ public static List<Expression> reconstructFunctionExpressions(
       FunctionExpression expression, List<List<Expression>> childExpressionsList) {
     List<Expression> resultExpressions = new ArrayList<>();
     for (List<Expression> functionExpressions : childExpressionsList) {
-      resultExpressions.add(
-          new FunctionExpression(
-              expression.getFunctionName(),
-              expression.getFunctionAttributes(),
-              functionExpressions));
+      resultExpressions.add(reconstructFunctionExpression(expression, functionExpressions));
     }
     return resultExpressions;
   }
 
   public static Expression reconstructFunctionExpression(
       FunctionExpression expression, List<Expression> childExpressions) {
-    return new FunctionExpression(
-        expression.getFunctionName(), expression.getFunctionAttributes(), childExpressions);
+    FunctionExpression copiedFunctionExpression = (FunctionExpression) expression.copy();
```

Comment:
remove the copy method in each `Expression` and then set viewPath path in this class.